server/alerts_email_via_python3.py

```diff
@@ -0,0 +1,273 @@
+<o:p></o:p></p></td></tr></table></div></td></tr></table><div class="Footer">
+<img src="https://github.com/securestate/king-phisher/raw/master/data/king-phisher-logo.png">
+</div>
+<p class=MsoNormal><o:p>&nbsp;</o:p></p></div></body></html>
+"""
+
+class Plugin(plugins.ServerPlugin):
+	authors = ['Austin DeFrancesco', 'Spencer McIntyre', 'Mike Stringer', 'Erik Daguerre']
+	classifiers = ['Plugin :: Server :: Notifications :: Alerts']
+	title = 'Campaign Alerts: via Python 3 SMTPLib'
+	description = """
+	Send campaign alerts via the SMTP Python 3 lib. This requires that users specify
+	their email through the King Phisher client to subscribe to notifications.
+	"""
+	homepage = 'https://github.com/securestate/king-phisher-plugins'
+
+	# Email accounts with 2FA, such as Gmail, will not work unless "less secure apps" are allowed
+	# Reference: https://support.google.com/accounts/answer/60610255
+	# Gmail and other providers require SSL on port 465, TLS will start with the activation of SSL
+	options = [
+		plugin_opts.OptionString(
+			name='smtp_host',
+			description='Location of SMTP server',
+			default='localhost'
+		),
+		plugin_opts.OptionInteger(
+			name='smtp_port',
+			description='Port used for SMTP server',
+			default=25
+		),
+		plugin_opts.OptionString(
+			name='smtp_email',
+			description='SMTP email address to send notifications from',
+			default=''
+		),
+		plugin_opts.OptionString(
+			name='smtp_password',
+			description='Password associated with smtp email',
+			default=''
+		),
+		plugin_opts.OptionBoolean(
+			name='ssl_tls',
+			description='Set connection to smtp server to use ssl/tls connection',
+			default=True
+		),
+		plugin_opts.OptionString(
+			name='email_jinja_template',
+			description='Password associated with smtp email',
+			default=''
+		),
+		plugin_opts.OptionString(
+			name='mail_options',
+			description='Additional mail options',
+			default=''
+		),
+	]
+	req_min_version = '1.3'
+	req_min_py_version = '3.0'
+	req_packages = {
+		'smtplib': has_smtp
+	}
+
+	def initialize(self):
+		signals.campaign_alert.connect(self.on_campaign_alert)
+		return True
+
+	def on_campaign_alert(self, table, alert_subscription, count):
+		user = alert_subscription.user
+		if not self.config['smtp_email']:
+			self.logger.debug("user {0} has no email address specified, skipping SMTP alert".format(user.id))
+			return False
+		if self.send_message(table, alert_subscription, count, self.config['smtp_email']):
+			return True
+		return False
+
+	def get_template_vars(self, table, alert_subscription, count):
+		campaign = alert_subscription.campaign
+		template_vars = {
+			'id': str(campaign.id),
+			'name': campaign.name,
+			'created': campaign.created,
+			'expiration': campaign.expiration,
+			'has_expired': campaign.has_expired,
+			'message_count': len(campaign.messages),
+			'visit_count': len(campaign.visits),
+			'credential_count': len(campaign.credentials),
+			'time': datetime.datetime.utcnow()
```

Comment:
I'm pretty sure `time` and `time.local` will be provided by the `TemplateEnvironmentBase`.